cap_frame_width: 1280
cap_frame_height: 720

# cap_frame_width: 1920
# cap_frame_height: 1080
